### ISSUE TITLE
Only Default Template works but not custom templates

### ISSUE THREAD

**Issue by lisandi** [it-projects-llc/website-addons#189]
Hi Ivan
We installed a custom theme and it has already done some debranding (concerning Odoo) but not all. Unfortunately the website_debranding stops with an error when the xpath expr. can't be located. 

Probably it would be a good idea to put this in an "if xpath expr" exists -> then get rid of it etc. 
Otherwise website_debranding can only be used on sites with the default or probably with Templates made by Odoo S.A. but not with those from other companies even they get sold on Odoo S.A. website or elsewhere or are freely available on git.

Version 10 and we tested with website_debranding 10.0.1.0.0

```
.. server/Software/models.py", line 1083, in _validate_fields
    raise ValidationError("%s\n\n%s" % (_("Error while validating constraint"), tools.ustr(e)))
ParseError: "Error while validating constraint

Element '<xpath expr="//div[@t-ignore='true']">' cannot be located in parent view

Error context:
View `layout_footer_copyright`
[view_id: 2559, xml_id: n/a, model: n/a, parent_id: 625]
None" while parsing None:4, near
<data inherit_id="website.layout_footer_copyright">
    <xpath expr="//div[@t-ignore='true']" position="replace">
    </xpath>
</data>
```

Thanks Andi

**Comment by arafatx** [comment]
what happen if you restart your server?

**Comment by yelizariev** [comment]
@lisandi correct. We can use python to apply such removing. One can find example here and make PR: https://github.com/it-projects-llc/misc-addons/blob/10.0/web_debranding/models/ir_ui_view.py

Sponsoring updates via bounty is welcome
